randovania/games/dread/json_data/Dairon.txt

```diff
@@ -2909,26 +2896,33 @@ Extra - asset_id: collision_camera_035
   > Pickup (Power Bomb Tank)
       Morph Ball
   > Ledge
-      Morph Ball and After Dairon - grapplepulloff1x2_003
+      All of the following:
+          After Dairon - Lake Puzzle Grapple Block
+          Any of the following:
+              Lay Bomb
+              Power Bombs ≥ 2 and Lay Power Bomb
 
 > Dock to Storm Missile Gate Room; Heals? False
   * Layers: default
   * Open Passage to Storm Missile Gate Room/Dock to Lake Puzzle Room
   > Ledge
       Any of the following:
-          Simple IBJ or Use Spin Boost
+          Walljump (Intermediate) or Simple IBJ or Use Spin Boost
           Flash Shift and Walljump (Beginner)
+          Speed Booster and After Dairon - Storm Missile Gate and Movement (Intermediate)
 
 > Ledge; Heals? False
   * Layers: default
-  > Event - Pull Grapple Block (grapplepulloff1x2_003)
+  > Event - Lake Puzzle Grapple Block
       All of the following:
           Grapple Beam
-          Lay Cross Comb or Lay Power Bomb
+          Any of the following:
+              # Power Bombs require desyncing the beam blocks
+              Lay Cross Comb or Lay Power Bomb
   > Above Grapple Block
       All of the following:
-          Morph Ball and After Dairon - grapplepulloff1x2_003
-          Cross Bomb or Gravity Suit or Space Jump
+          Morph Ball and After Dairon - Lake Puzzle Grapple Block
+          Cross Bomb or Gravity Suit or Use Spin Boost
```

Comment:
How does spinboost reach the water tunnel? It looks like it's too far to reach

Reply:
You can just fall through the crumble blocks then use spin boost to grab the ledge